CHANGES.rst

```diff
@@ -37,6 +37,12 @@ master_background
 
 - Updated step arguments in the documentation. [#4723]
 
+mrs_imatch
+-----------------
```

Comment:
Underline length must match number of characters in text.

Reply:
Done

Reply:
I *think* it's the case that it's OK to have the underline be *longer* than the corresponding text, but not *shorter*. Of course it doesn't hurt to always make them match, just in case.